### ISSUE TITLE
Web UI files

### ISSUE THREAD

**Issue by Frando** [Studio-Link-v2/backend#38]
We're thinking about using/customizing studiolink for out usecase, which is mobile outside broadcast link back to the studio (likely with Raspberry Pi over 4G). For that we'd like to customize the UI. Do you have build scripts that build the js.h, css.h, index_html.h etc from the actual sources? Esp. for the JS, as I'd assume that there's build steps involved (at least it's several files concatenated).

Basically, we'd like to extract the websocket API part out of the JS that's in js.h. Before doing it by hand, I wanted to push this issue as I guess that you have the source and build files somewhere.

**Comment by Frando** [comment]
So I can parse the JS out of the c files. However, it would be much nicer to work on the original source files. Could you publish those together with the precompiler that generates the header files here in the repo?
